lib/testUtils/readJSONFile.mjs

```diff
@@ -0,0 +1,9 @@
+import * as fs from 'node:fs';
+
+/**
+ * @param {string | URL} filePath
+ * @returns {unknown}
+ */
+export default function readJSONFile(filePath) {
+	return JSON.parse(fs.readFileSync(filePath, 'utf8'));
```

Comment:
question

Can't you do 
```
import configExtendingAnotherExtend from './fixtures/config-extending-another-extend.json' assert { type : 'json' };
```
instead?

Reply:
This is an experimental syntax, that [will change](https://twitter.com/robpalmer2/status/1639014193369354240) to another syntax.

Reply:
See https://github.com/stylelint/stylelint/pull/6941#discussion_r1236012487

Jest supports Import Assertions, while ESLint doesn't.